src/server/models/listItem.ts

```diff
@@ -51,74 +44,54 @@ async function createCountry(country: string): Promise<Country> {
 async function getPlaceGeoPoint(props: {
   countryName?: string;
   text?: string;
-}): Promise<Point | undefined> {
+}): Promise<Point> {
   const { countryName, text } = props;
 
   if (text === undefined || countryName === undefined) {
-    return undefined;
+    return [0.0, 0.0];
   }
 
   try {
-    const place = await geoLocatePlaceByText(`${text}, ${countryName}`);
-    return place?.Geometry?.Point ?? undefined;
+    const { Geometry } = await geoLocatePlaceByText(`${text}, ${countryName}`);
+
+    if (Geometry.Point !== undefined) {
+      return Geometry.Point;
+    }
+
+    return [0.0, 0.0];
   } catch (error) {
-    return undefined;
+    return [0.0, 0.0];
   }
 }
 
 async function createAddressGeoLocation(
   item: LawyersFormWebhookData | CovidTestSupplierFormWebhookData
-): Promise<number | boolean> {
+): Promise<number> {
   let address: string;
 
   if ("organisationDetails" in item) {
     address = `
-      ${item.organisationDetails.addressLine1}, 
-      ${item.organisationDetails.addressLine2 ?? ""}, 
-      ${item.organisationDetails.city} - 
-      ${item.organisationDetails.country} - 
+      ${item.organisationDetails.addressLine1},
+      ${item.organisationDetails.addressLine2 ?? ""},
+      ${item.organisationDetails.city} -
+      ${item.organisationDetails.country} -
       ${item.organisationDetails.postcode}
     `;
   } else {
     address = `
-      ${item.addressLine1}, 
-      ${item.addressLine2 ?? ""}, 
-      ${item.city} - 
-      ${item.country} - 
+      ${item.addressLine1},
+      ${item.addressLine2 ?? ""},
+      ${item.city} -
+      ${item.addressCountry} -
       ${item.postcode}
     `;
   }
 
-  const location = await geoLocatePlaceByText(address);
-  const point = location?.Geometry?.Point;
-
-  if (isArray(point)) {
-    return await rawInsertGeoLocation(point);
-  }
+  const { Geometry } = await geoLocatePlaceByText(address);
 
-  return false;
-}
+  const point = Geometry.Point ?? [0.0, 0.0];
```

Comment:
Fair shout. I've cleaned this up to get the services to consistently return the `Point` and to avoid having to do any checks or destructuring when implementing